packages/utils/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json.schemastore.org/tsconfig",
-  "extends": "@vertexwebsdk/build/tsconfig",
+  "extends": "@vertexwebsdk/build/tsconfig-web",
```

Comment:
Any concern that this config would allow for methods that are unsafe for NodeJS? Not sure if we're using it in any node projects, but our rollup config specifies `isMultiplatform: true`, and this would allow me to use `window` functions that'd cause issues there